Flaky tests

The following test can fails in non deterministic manner with error reported below

https://github.com/seqeralabs/wave/blob/ac900dad00b4925cfa3795510bd1f0b89bb227df/src/test/groovy/io/seqera/wave/service/builder/cache/LocalBuildStoreTest.groovy#L21-L21

```
io.micronaut.context.exceptions.BeanInstantiationException: Bean definition [io.seqera.wave.core.RegistryProxyService] could not be loaded: Error instantiating bean of type  [io.seqera.wave.storage.RedisStorage]

Message: Unable to connect to localhost/<unresolved>:49173
Path Taken: RegistryProxyService.setStorage([Storage storage]) --> RedisStorage.redisConn
	at app//io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1938)
	at app//io.micronaut.context.DefaultApplicationContext.initializeContext(DefaultApplicationContext.java:237)
	at app//io.micronaut.context.DefaultBeanContext.readAllBeanDefinitionClasses(DefaultBeanContext.java:3453)
	at app//io.micronaut.context.DefaultBeanContext.finalizeConfiguration(DefaultBeanContext.java:3883)
	at app//io.micronaut.context.DefaultBeanContext.start(DefaultBeanContext.java:329)
	at app//io.micronaut.context.DefaultApplicationContext.start(DefaultApplicationContext.java:183)
	at app//io.micronaut.context.ApplicationContextBuilder.start(ApplicationContextBuilder.java:244)
	at app//io.micronaut.context.ApplicationContext.run(ApplicationContext.java:179)
	at app//io.micronaut.context.ApplicationContext.run(ApplicationContext.java:161)
	at app//io.seqera.wave.service.builder.cache.RedisBuildStoreTest.setup(RedisBuildStoreTest.groovy:28)
Caused by: io.micronaut.context.exceptions.BeanInstantiationException: Error instantiating bean of type  [io.seqera.wave.storage.RedisStorage]

Message: Unable to connect to localhost/<unresolved>:49173
Path Taken: RegistryProxyService.setStorage([Storage storage]) --> RedisStorage.redisConn
	at app//io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2363)
	at app//io.micronaut.context.DefaultBeanContext.createAndRegisterSingletonInternal(DefaultBeanContext.java:3281)
	at app//io.micronaut.context.DefaultBeanContext.createAndRegisterSingleton(DefaultBeanContext.java:3267)
	at app//io.micronaut.context.DefaultBeanContext.getBeanForDefinition(DefaultBeanContext.java:2820)
	at app//io.micronaut.context.DefaultBeanContext.getBeanInternal(DefaultBeanContext.java:2782)
	at app//io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1638)
	at app//io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:1933)
	at app//io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForField(AbstractInitializableBeanDefinition.java:1487)
	at app//io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2336)
	at app//io.micronaut.context.DefaultBeanContext.createAndRegisterSingletonInternal(DefaultBeanContext.java:3281)
	at app//io.micronaut.context.DefaultBeanContext.createAndRegisterSingleton(DefaultBeanContext.java:3267)
	at app//io.micronaut.context.DefaultBeanContext.getBeanForDefinition(DefaultBeanContext.java:2820)
	at app//io.micronaut.context.DefaultBeanContext.getBeanInternal(DefaultBeanContext.java:2782)
	at app//io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1638)
	at app//io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:1933)
	at app//io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForMethodArgument(AbstractInitializableBeanDefinition.java:1047)
	at app//io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2336)
	at app//io.micronaut.context.DefaultBeanContext.createAndRegisterSingletonInternal(DefaultBeanContext.java:3281)
	at app//io.micronaut.context.DefaultBeanContext.loadContextScopeBean(DefaultBeanContext.java:2664)
	at app//io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1932)
	... 9 more
Caused by: io.lettuce.core.RedisConnectionException: Unable to connect to localhost/<unresolved>:49173
	at app//io.lettuce.core.RedisConnectionException.create(RedisConnectionException.java:78)
	at app//io.lettuce.core.RedisConnectionException.create(RedisConnectionException.java:56)
	at app//io.lettuce.core.AbstractRedisClient.getConnection(AbstractRedisClient.java:330)
	at app//io.lettuce.core.RedisClient.connect(RedisClient.java:216)
	at app//io.lettuce.core.RedisClient.connect(RedisClient.java:201)
	at app//io.micronaut.configuration.lettuce.AbstractRedisClientFactory.redisConnection(AbstractRedisClientFactory.java:88)
	at app//io.micronaut.configuration.lettuce.DefaultRedisClientFactory.redisConnection(DefaultRedisClientFactory.java:56)
	at app//io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2336)
	... 28 more
Caused by: io.netty.channel.unix.Errors$NativeIoException: readAddress(..) failed: Connection reset by peer

```

--- a/src/test/groovy/io/seqera/wave/test/RedisTestContainer.groovy
+++ b/src/test/groovy/io/seqera/wave/test/RedisTestContainer.groovy
@@ -17,6 +17,7 @@ trait RedisTestContainer {
     @Shared
     static GenericContainer redisContainer = new GenericContainer(DockerImageName.parse("redis:7.0.4-alpine"))
             .withExposedPorts(6379)
+            .waitingFor(Wait.forLogMessage(".*Ready to accept connections.*\\n", 1));
 
 
     void restartRedis(){

